java/memory/memory-netty/pom.xml

```diff
@@ -52,6 +53,55 @@
     </dependency>
   </dependencies>
 
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-shade-plugin</artifactId>
+        <version>3.1.0</version>
+        <executions>
+          <execution>
+            <phase>package</phase>
+            <goals>
+              <goal>shade</goal>
+            </goals>
+            <configuration>
+              <shadedArtifactAttached>false</shadedArtifactAttached>
+              <dependencyReducedPomLocation>${project.build.directory}/dependency-reduced-pom.xml</dependencyReducedPomLocation>
+              <transformers>
+                <transformer implementation="org.apache.maven.plugins.shade.resource.ServicesResourceTransformer"/>
+              </transformers>
+              <relocations>
+                <relocation>
+                  <pattern>io.netty</pattern>
+                  <shadedPattern>shaded.io.netty</shadedPattern>
```

Comment:
@scampi @GeorgeAp we add netty as shaded here.

Reply:
is this needed because of the netty dependency to get the benchmark ? https://github.com/sirensolutions/siren-platform/pull/7058#discussion_r1687661177

If so, then let's just remove that benchmark in a separate commit. It doesn't make much sense to make things more complex for a benchmark that isn't run. If one day we want to compare what we have against netty, then we can look at the commit.